### PR TITLE
Adding and modifying text files for TLEs catalogue downloader

### PR DESCRIPTION
**Pull request by bxaia**
This PR modifies some text files and adds more text files for TLE catalogue downloader.

Jira: [MT-1606](https://skaafrica.atlassian.net/browse/MT-1606)

### CODE REVIEW

**Comment by bxaia** [comment]
@ajoubertza, @adriaanph, @mamkhari  please review.

File: catalogues/intelsat.txt [hunk 1]
```diff
@@ -0,0 +1,441 @@
+INTELSAT 26         [PM]
+1 24732C 97007A   20189.09375000  .00000108  00000-0  00000-0 0  1895
+2 24732   9.0673  42.8766 0007644  54.0012 286.0148  1.00270895    13
+INTELSAT 26             
+1 24732C 97007A   20191.00000000  .00000108  00000-0  00000-0 0  1917
+2 24732   9.0693  42.8846 0004866  66.1278 242.0371  1.00266430    15
+INTELSAT 26             
+1 24732C 97007A   20195.00000000  .00000108  00000-0  00000-0 0  1955
+2 24732   9.0754  42.8484 0004802  68.8579 243.2857  1.00267809    17
```
Comment (ajoubertza) [inline catalogues/intelsat.txt:9]: @adriaanph In these files we have the same satellite 3 times.  The first has "[PM]" in the name, while the remaining two have identical names "INTELSAT 26".  I'm not sure which target you will get in `kat.sources`.    It is a dict, so you won't get duplicate names.

**Review by adriaanph — APPROVED** [review]
this looks 100%, thanks.

**Review by ajoubertza — APPROVED** [review]
Good.
We just need to be careful about when we merge this.  The downloader config on site has be updated at the same time, otherwise some of the old files will come back.  That's what resulted in the conflicts that have appeared since yesterday (daily updates pushed from site)

File: catalogues/intelsat.txt (hunk 1 above)
Comment (ajoubertza) [reply catalogues/intelsat.txt:9]: The `kat.sources` object is a `katpoint.Catalogue` object, and it [says](https://github.com/ska-sa/katpoint/blob/65f7a18083b3d44c8abc69b5f5d7723ed8c1509d/katpoint/catalogue.py#L69-L70) this about duplicates:
```
The most recently added target with the specified name is returned
```

Loading this current set of targets reports the following duplicates (and will print this big blob of text each time someone uses katuilib).

```
WARNING:katpoint.catalogue:Found different targets with same name(s) 'J0534+2200' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'J0835-4510' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'J1745-2900' in catalogue
WARNING:katpoint.catalogue:5 of 24 TLE set(s) are outdated, probably making them inaccurate for use right now
WARNING:katpoint.catalogue:Worst case: TLE epoch for 'COSMOS 2545 (760)' is 242 days in past, should be <= 21 for deep-space model
WARNING:katpoint.catalogue:3 of 147 TLE set(s) are outdated, probably making them inaccurate for use right now
WARNING:katpoint.catalogue:Worst case: TLE epoch for 'INTELSAT 907' is 28 days in past, should be <= 21 for deep-space model
WARNING:katpoint.catalogue:Found different targets with same name(s) 'INTELSAT 26' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'GALAXY 25' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'INTELSAT 5' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'GALAXY 11' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'INTELSAT 9' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'INTELSAT 12' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'INTELSAT 1R' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'INTELSAT 10' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'INTELSAT 901' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'INTELSAT 902' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'INTELSAT 904' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'INTELSAT 905' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'GALAXY 3C' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'INTELSAT 906' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'INTELSAT 907' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'GALAXY 12' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'GALAXY 23' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'GALAXY 13' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'INTELSAT 10-02' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'GALAXY 28' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'GALAXY 14' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'GALAXY 15' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'GALAXY 16' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'GALAXY 17' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'INTELSAT 11' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'HORIZONS 2' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'GALAXY 18' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'INTELSAT 25' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'GALAXY 19' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'INTELSAT 14' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'INTELSAT 15' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'INTELSAT 16' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'INTELSAT 17' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'INTELSAT NEW DAWN' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'INTELSAT 18' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'INTELSAT 22' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'INTELSAT 19' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'INTELSAT 20' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'INTELSAT 21' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'INTELSAT 23' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'INTELSAT 30' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'INTELSAT 34' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'INTELSAT 31' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'INTELSAT 36' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'INTELSAT 33E' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'INTELSAT 32E' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'INTELSAT 35E' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'INTELSAT 37E' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'AZERSPACE 2' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'HORIZONS 3E' in catalogue
WARNING:katpoint.catalogue:Found different targets with same name(s) 'INTELSAT 39' in catalogue
```

Comment (adriaanph) [reply catalogues/intelsat.txt:9]: 1) this change replaces glo-ops.txt with glo.txt, and gps-ops.txt with gps.txt. the "-ops" files had all the nonoperational satellites filtered out which is still included in the new files. that's why the "outdated" warnings appear. i don't see this as a problem.

2) satellites that are FILTERED OUT by their *_keep.txt files must not cause warning messages to appear. please address this issue.

3) it seems __only the intelsat.txt supplemental file__ contains two entries for each satellite with the same name. these entries are 3 days apart and listed with the most recent one, the one for the current day, last. with the current implementation the catalogue will therefore retain the most current TLE which is exactly what is desired. the warning messages are unfortunate, but should probably just be left as it is.

4) the intelsat TLE's marked with "[PM]" can be ignored, unless specifically included in intelsat_keep.txt and then the user should know what (s)he's up to.

Comment (ajoubertza) [reply catalogues/intelsat.txt:9]: > 2\. satellites that are FILTERED OUT by their *_keep.txt files must not cause warning messages to appear. please address this issue.

This is a limitation of katpoint.  CAM's [`katcorelib.conf.SourceConfig.get_catalogue`](https://github.com/ska-sa/katcorelib/blob/d30c69a414b6387e3cd9f8a6c802bd6513f7a3df/katcorelib/conf.py#L59) gives katpoint the whole TLE file which it loads, along with warnings.  Once the targets are loaded, CAM can filter them out easily.

I suggest you raise an issue in [katpoint](https://github.com/ska-sa/katpoint) - if it had an optional filter parameter on the [`add_tle`](https://github.com/ska-sa/katpoint/blob/10ccb9de1027b7ab618ed0e0541b0c2acbe064c6/katpoint/catalogue.py#L468) method, then we could give it a short list to consider.

**Review by ajoubertza — APPROVED** [review]
Thanks for readme @bxaia.  I added some more details that came to mind.

**Comment by bxaia** [comment]
> Thanks for readme @bxaia. I added some more details that came to mind.

thank you

**Review by adriaanph — APPROVED** [review]
i'm very happy with this.

**Pull request merged**